godot-core/src/builtin/variant/variant_traits.rs

```diff
@@ -49,12 +49,15 @@ pub trait ToVariant {
 
 // ----------------------------------------------------------------------------------------------------------------------------------------------
 
-#[derive(Eq, PartialEq, Debug)]
-pub struct VariantConversionError;
-/*pub enum VariantConversionError {
+#[derive(Debug, Eq, PartialEq)]
+//pub struct VariantConversionError;
+pub enum VariantConversionError {
```

Comment:
Could you use the previous `#[derive]` order and delete the commented-out code?
```suggestion
#[derive(Eq, PartialEq, Debug)]
pub enum VariantConversionError {
```